proof/refine/AARCH64/ArchKHeap_R.thy

```diff
@@ -15,7 +15,7 @@ context Arch begin arch_global_naming
 named_theorems KHeap_R_assms
 
 lemma getObject_inv_vcpu[wp]:
-  "\<lbrace>P\<rbrace> getObject l \<lbrace>\<lambda>(rv :: ArchStructures_H.vcpu). P\<rbrace>"
+  "\<lbrace>P\<rbrace> getObject l \<lbrace>\<lambda>(_ :: ArchStructures_H.vcpu). P\<rbrace>"
```

Comment:
Incredibly minor but just to clarify, I was originally more commenting on the brackets not being needed around the type annotation here and that we normally don't have spaces around `::` when it's inside a lambda like that.

Reply:
oh, got it, will add to list

Reply:
Looks good, thanks!